scripts/create-migration.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
```

Comment:
You can use the [ory cli](https://github.com/ory/cli) to achieve this:

```
ory dev pop migration create 
Error: accepts 2 arg(s), received 0
Usage:
  ory dev pop migration create [destination] [name] [flags]

Flags:
  -d, --dialect string   Choose a dialect from: [sqlite cockroach mysql postgres]
  -h, --help             help for create

accepts 2 arg(s), received 0
```

:)

Reply:
Cool! Was not aware this existed. Removed my bash script and re-created those files w/ the CLI.